src/api/signals/summaries.py

```diff
@@ -21,14 +19,10 @@ def update_summaries_on_delete_transect_method(sender, instance, *args, **kwargs
 
     sample_unit = instance.sample_unit
     sample_unit.delete()
-    submit_job(
-        5,
-        True,
-        update_summary_cache,
-        project_id=project.pk,
-        sample_unit=instance.protocol,
-        timestamp=timezone.now(),
-    )
+    try:
+        add_project_to_queue(project.pk)
+    except Exception as e:
+        print(f"Failed to queue summary update for project {project.pk}: {e}")
```

Comment:
Hmmm -- wanting something other than a `print()` when things go wrong. Use logging framework to email me in prod, or output to Cloudwatch?

Reply:
This would be the equivalent of submit_job not working (currently a Cloudwatch log).  I can change this to notify if you want.